pages/en/lb3/Authentication-authorization-and-permissions.md

```diff
@@ -68,55 +68,87 @@ module.exports = function enableAuthentication(server) {
+Normally you will implement at least one custom user model, extending the built-in `User` model, as described in [Using built-in models](Using-built-in-models.html#user-model).
 
-Usually you don't need to extend or customize the built-in models `Role`, `RoleMapping`, and `ACL`. Just make sure they are declared in the `model-config.json` configuration file, or in case you don't require either to customize the `AccessToken` model that you pass a datasource to the `enableAuth()` method as follows:
+Usually you don't need to extend or customize the built-in models `Role`, `RoleMapping`, and `ACL`. Just make sure they are declared in the `model-config.json` configuration file, or if you don't need to customize the `AccessToken` model, pass a datasource to the `enableAuth()` method as follows:
 
 ```javascript
 server.enableAuth({ datasource: 'db' });
 ```
 
-{% include note.html content="
-Passing a `datasource` to the `enableAuth()` method as shown here will let LoopBack take care of attaching any built-in models required by the access control feature, which is suitable for most applications.
+{% include note.html content="Passing a `datasource` to the `enableAuth()` method as shown here will let LoopBack take care of attaching any built-in models required by the access control feature, which is suitable for most applications.
 " %}
 
-{% include tip.html content="
-Whether you can use the built-in `AccessToken` model or create a custom accessToken model extending the built-in model depends on whether you plan to use one or several user models extending the built-in `User` model. Both cases are covered in the next two sections.
+{% include tip.html content="Whether you can use the built-in `AccessToken` model or create a custom model that extends `AccessToken` depends on whether you plan to use one or several user models extending the built-in `User` model, as described in the following sections.
 " %}
 
-#### Access control with a single user model
+#### Access control with a single user model and built-in AccessToken model
 
-If your application leverages only one type of user extending the built-in `User` model (which should the case in a majority of configurations), you have little further configuration to do.
+In the most common scenario, an application uses only one model that extends the built-in `User` model
+and uses the built-in `AccessToken` model.  In this case, you need to change the "belongsTo" relation of the built-in `AccessToken` model to refer to your custom User model.  To do this, edit `server/model-config.json` file as follows:
 
-1.  Rely on the built-in `AccessToken` model.  
-2.  Make sure your custom user model implements a hasMany relation with the AccessToken model as follows:
+{% include code-caption.html content="server/model-config.json" %}
+```json
+{
+  // ...
+  "AccessToken": {
+    "dataSource": "db",
+    "public": false,
+    "relations": {
+      "user": {
+        "type": "belongsTo",
+        "model": "user",
```

Comment:
BTW : while being a good practice in any case: is this absolutely required for single user model scenario since you've fixed the possibility of the missing relation with user [here](https://github.com/strongloop/loopback/pull/3227/files#diff-5d81af95449574544e7317f3eda2badeR695) ?